.vaunt/config.yaml

```diff
@@ -0,0 +1,89 @@
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition: "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-19 20:00:00
+                        & labels in {'2024-tidb-docs-dash','contribution'}"
+    - achievement:
+          name: Docs Dash Issue Contribution
+          icon: <icon_url>
+          description: Commented on an issue created during the docs dash event with the labl 2024-tidb-docs-dash and get moderator reaction!
+          triggers:
+              - trigger:
+                    actor: author
+                    action: issue_comment
+                    condition: "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & labels in {'2024-tidb-docs-dash'}
+                        & emojis in ['THUMBS_UP']
+                        & reactors in ['rpaik', 'lilin90', 'qiancai', 'hfxsd', 'ran-huang', 'Oreoxmt','Yan-yyCL','dveeden']"
+    - achievement:
+          name: PR Submitted
+          icon: <icon_url>
+          description: Submit any PR during the docs dash event between January 9th - 12th 2024!
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition: "created_at >= 01-09-2024 08:00:00 & created_at <= 01-11-2024
+                         & labels in !['2024-tidb-docs-dash']
+                         & labels in ['contribution']"
+    - achievement:
+          name: PR Merged
+          icon: <icon_url>
+          description: Have any PR merged during the docs dash event
+          triggers:
+              - trigger:
+                    actor: author
+                    action: pull_request
+                    condition: "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & merged = true & merged_at <= 2024-01-19 20:00:00
+                        & labels in !['2024-tidb-docs-dash']
+                        & labels in ['contribution']"
+    - achievement:
+          name: Issue Contribution
+          icon: <icon_url>
+          description: Commented on any issue and get moderator reaction!
+          triggers:
+              - trigger:
+                    actor: author
+                    action: issue_comment
+                    condition: "created_at >= 01-09-2024 08:00:00 & created_at <= 01-12-2024 07:59:00
+                        & emojis in ['THUMBS_UP']
+                        & reactors in ['rpaik', 'lilin90', 'qiancai', 'hfxsd', 'ran-huang', 'Oreoxmt','Yan-yyCL','dveeden']
+                        & labels in !['2024-tidb-docs-dash']"
+    - achievement:
+          name: Docs Dash Bonus Points
+          icon: <icon_url>
+          description: Bbonus points for substantial creative contributions!
+          triggers:
```

Comment:
Does it mean the bonus points for a valuable issue comment are the same as those for a valuable PR? or Can we make them different?  Valuable PRs usually require more effort and deserve more bonus points, perhaps double bonus points for PRs?

Reply:
I can update the config to have these be two different achievements - 1 for pull requests and another for issues. 

From the Vaunt POV - a single achievement accomplishes the same thing as making the separate since the trigger is what awards the achievement, not a points system. 

Alternatively, we can remove the bonus achievement from the config file and we can manage any bonus achievements through our API manual after points have been calculated.

Reply:
@qiancai that's a good point. 

@elewis787 maybe it's easier to have two separate achievements? "Docs Dash Bonus Points Issues" and "Docs Dash Bonus Points PRs"?

Reply:
Sounds good. I will update the config shortly!

Reply:
I have updated the PR to split the bonus achievements!